### ISSUE TITLE
Support Flutter 2.0

### ISSUE THREAD

**Issue by yamauchieduardo** [subosito/flutter-action#85]
The action support Flutter 2.0? 

If yes, should be better to update the example.

**Comment by Andrewngabriel** [comment]
You can use Flutter 2.0 by switching to using the master version of the action
```
steps:
  - uses: actions/checkout@v2
  - uses: subosito/flutter-action@master
    with:
      channel: master
  - run: flutter pub get
```

**Comment by guidezpl** [comment]
FYI this is now available with `subosito/flutter-action@v1.5`
